meta-facebook/op2-op/src/platform/plat_util.h

```diff
@@ -24,6 +24,7 @@
 #define IPMI_EVENT_OFFSET_SYS_E1S_P12V_FAULT 0x83
 #define IPMI_EVENT_OFFSET_SYS_E1S_P3V3_FAULT 0x84
 #define IPMI_EVENT_OFFSET_SYS_INA233_ALERT 0x86
+#define IPMI_EVENT_OFFSET_SYS_EXPA_CLOCL_BUFFER 0x88
```

Comment:
Did u intend xxx_CLOCK_xxx instead of xxx_CLOCL_xxx?

Reply:
> Did u intend xxx_CLOCK_xxx instead of xxx_CLOCL_xxx?

Can you please check this comment. Thanks

Reply:
Hi jagpalgill,
    Thanks for your correction, I modify this naming.